torch/_dynamo/variables/functions.py

```diff
@@ -59,13 +62,13 @@ def wrap_bound_arg(tx, val, options, source=None):
     elif istensor(val):
         from torch._dynamo.variables.builder import VariableBuilder
 
-        return VariableBuilder(tx, source=source, **options)(val)
+        return VariableBuilder(tx, source=source)(val).add_options(options)
```

Comment:
VariableBuilder signature is (tx, source). I guess, earlier the options were always empty and therefore this bug stayed hidden.